package/kedro_viz/integrations/kedro/sqlite_store.py

```diff
@@ -42,10 +43,22 @@ def to_json(self) -> str:
         """Returns session_store information in json format after converting PosixPath to string"""
         session_dict = {}
         for key, value in self.data.items():
-            if _is_json_serializable(value):
+            # Get branch info through git_sha
+            if key == "git":
+                sha = [v for k, v in value.items() if k == "commit_sha"]
```

Comment:
Why do you need to loop through the values? Can't you just do `value.get("commit_sha")`?

Reply:
And what would happen if we don't have any "git" data at all or if the process can't complete for some reason?

Reply:
i made changes to the code. if there's no git then branch is not added at all.